src/test/robot/api/storage/data/create_and_read_events.js

```diff
@@ -0,0 +1,45 @@
+PreconditionLoad
+	.config({
+		"item" : {
+			"type" : "data",
+			"output" : {
+				"path" : "test"
+			},
+            "data" : {
+                "size" : "1000KB"
+            }
+		},
+	    "load" : {
+        	"op" : {
+            	"limit" : {
+                	"count" : 1000
+            	}
+        	}
+    	},
+        "storage" : {
```

Comment:
move to the shared config or command line arguments (better)

Reply:
done